_sass/all.scss

```diff
@@ -39,12 +39,13 @@ body {
   margin-bottom: 50px;
   position: relative;
 }
+
 h2 {
-  margin-bottom: 20px;
-  margin-top: 20px;
-  font-weight: 700;
-  text-align: center;
-  color: var(--color-p-text);
```

Comment:
in dark theme , color of h2 headings are not changing. you need to include this for it